src/groupByKey.ts

```diff
@@ -2,6 +2,17 @@ interface Item {
   [key: string]: any;
 }
 
-export function groupByKey(items, field) {
-  // write code here;
+export function groupByKey(items: Item[], field: string): Item {
```

Comment:
```suggestion
export function groupByKey(items: Item[], field: keyof Item): Item {
```
Мы передаем не просто строку, а именно ключи из объекта, поэтому правильней будет это явно указать

Reply:
Слушно, дякую! зараз виправлю